src/Uno.UI.Runtime.Skia.X11/X11NativeOverlappedPresenter.cs

```diff
@@ -66,9 +66,9 @@ public void Minimize(bool activateWindow)
 
 	public void SetBorderAndTitleBar(bool hasBorder, bool hasTitleBar)
 	{
-		// Border doesn't seem to do anything except show the title bar even if !hasTitleBar, which is fine for now,
-		// since it doesn't do anything on WinUI either.
-		// X11Helper.SetMotifWMDecorations(x11Window, hasBorder, (IntPtr)MotifDecorations.Border);
+		// On some WMs, Border doesn't seem to do anything except show the title bar even if !hasTitleBar.
+		// However, not setting it would cause the window border to completely disappear on other environments.
```

Comment:
Let's specify explicitly which environments, and point to the original issue.

@Darsh0307 let's add this particular case to the test suite.